class/defaults.yml

```diff
@@ -5,6 +5,9 @@ parameters:
     namespace: syn-nfs-subdir-external-provisioner
     release_name: ${_instance}
     helm_values:
+      image:
+        # Workaround until https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/pull/252 has been merged
+        repository: registry.k8s.io/sig-storage/nfs-subdir-external-provisioner
```

Comment:
Do we need to add the workaround in the component, or would it be sufficient to have this in the Commodore global defaults until the Helm chart is updated?

Reply:
We can. But I fear, that this is more likely to be forgotten if it is in the Commodore global defaults, than in the component.